Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/GGVPointer.cs

```diff
@@ -240,6 +242,8 @@ public void OnInputDown(InputEventData eventData)
                             c.SourceDownIds.Add(eventData.SourceId);
                         }
                         InputSystem.RaisePointerDown(this, selectAction, Controller.ControllerHandedness);
+
+                        gazeProvider.GazePointer.IsFocusLocked = (gazeProvider.GazePointer.Result?.Details.Object != null);
```

Comment:
Doesn't the input system already lock / unlock pointers? Or, is the problem that the GGVPointer gets locked, but not the gaze pointer?

I'd recomment adding a comment explaining we we are locking / unlocking here.

Reply:
> Or, is the problem that the GGVPointer gets locked, but not the gaze pointer?

Yeah, exactly this. I'll add a comment.